packages/ui/src/components/Visualization/Canvas/CanvasFormTabs.tsx

```diff
@@ -0,0 +1,118 @@
+import { ToggleGroup, ToggleGroupItem } from '@patternfly/react-core';
+import { FunctionComponent, useCallback, useContext, useEffect, useMemo, useRef, useState } from 'react';
+import { EntitiesContext } from '../../../providers/entities.provider';
+import { SchemaBridgeProvider } from '../../../providers/schema-bridge.provider';
+import { getUserUpdatedPropertiesSchema, isDefined, setValue } from '../../../utils';
+import { CustomAutoForm, CustomAutoFormRef } from '../../Form/CustomAutoForm';
+import { DataFormatEditor } from '../../Form/dataFormat/DataFormatEditor';
+import { LoadBalancerEditor } from '../../Form/loadBalancer/LoadBalancerEditor';
+import { StepExpressionEditor } from '../../Form/stepExpression/StepExpressionEditor';
+import { UnknownNode } from '../Custom/UnknownNode';
+import './CanvasFormTabs.scss';
+import { CanvasNode } from './canvas.models';
+
+interface CanvasFormTabsProps {
+  selectedNode: CanvasNode;
+}
+
+export const CanvasFormTabs: FunctionComponent<CanvasFormTabsProps> = (props) => {
+  const entitiesContext = useContext(EntitiesContext);
+  const divRef = useRef<HTMLDivElement>(null);
+  const formRef = useRef<CustomAutoFormRef>(null);
+  const omitFields = useRef(props.selectedNode.data?.vizNode?.getBaseEntity()?.getOmitFormFields() || []);
+  const modes = ['Default', 'Modified'];
```

Comment:
Could this be an `enum` instead? I think if we use an `enum` + `ToggleGroup`, whenever the selected mode changes, will reload the form which is positive in this case that we want to regenerate the `<AutoForm>`